README.md

```diff
@@ -1 +1,40 @@
-# webpack
\ No newline at end of file
+# Leaderboard
+
+> 
+
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
```

Comment:
- [ ] Please use this [template(https://github.com/microverseinc/readme-template) as a reference to make your README more professional. Ensure that you have all the main headers like "Getting Started".